### PR TITLE
Create new Orb Indicator component

### PR DESCRIPTION
**Pull request by maxplt**
# Jira Issue: [IP-92](https://hedvig.atlassian.net/jira/software/projects/IP/boards/19?selectedIssue=IP-92)

## What?
Create Orb Indicator components without semantic-ui

## Why?
For less external libraries

## Video

https://user-images.githubusercontent.com/89198006/137849388-ed60be47-707a-4fa3-9a42-78f8529ffe38.mov

## Optional checklist
- [ ] Updated `src/changelog.ts`
- [ ] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

File: shared/@hedvig-ui/OrbIndicator/orb-indicator.tsx [hunk 1]
```diff
@@ -1,59 +1,96 @@
...
+  green: '#21ba45',
+  teal: '#00b5ad',
+  blue: '#2185d0',
+  violet: '#6435c9',
+  purple: '#a333c8',
+  pink: '#e03997',
+  brown: '#a5673f',
+  grey: '#767676',
+  black: '#1b1c1d',
+}
```
Comment (rasmusguterstam) [inline shared/@hedvig-ui/OrbIndicator/orb-indicator.tsx:50]: Do we really need all of these? And wouldn't it be nice if we could use the danger/warning/success etc. from theme instead? 😄

File: shared/@hedvig-ui/OrbIndicator/orb-indicator.tsx [hunk 2]
```diff
@@ -1,59 +1,96 @@
...
+  'violet',
+  'purple',
+  'pink',
+  'brown',
+  'grey',
+  'black',
+] as const
 
-type Flag = 'GREEN' | 'AMBER' | 'RED'
+export type SizeProp = typeof OrbSizes[number]
```
Comment (rasmusguterstam) [inline shared/@hedvig-ui/OrbIndicator/orb-indicator.tsx:30]: Why not just `export type OrbColors = 'red' | 'orange' ...`? 😮

File: shared/@hedvig-ui/OrbIndicator/orb-indicator.tsx [hunk 3]
```diff
@@ -1,59 +1,96 @@
...
+}
+
+const sizes = {
+  mini: '11px',
+  tiny: '14px',
+  small: '21px',
+  large: '41px',
+  big: '55px',
+  huge: '110px',
+  massive: '220px',
```
Comment (rasmusguterstam) [inline shared/@hedvig-ui/OrbIndicator/orb-indicator.tsx:59]: 😅

File: shared/@hedvig-ui/OrbIndicator/orb-indicator.tsx (hunk 1 above)
Comment (maxplt) [reply shared/@hedvig-ui/OrbIndicator/orb-indicator.tsx:50]: I just took these colors from semantic component

**Review by rasmusguterstam — APPROVED** [review]
Nice one! But I'm not really sure we want to introduce unlikely sizes and colors. Not that they are wrong in themselves, but it opens up for the possibility that people will use them in an unintended way (i.e. using 'green' to illustrate success instead of just using `theme.success`).

I'll approve, but think about it and challenge it if you want to! 😄

**Review by rasmusguterstam — APPROVED** [review]

**Pull request merged**